user_guide_src/source/installation/upgrade_430.rst

```diff
@@ -80,6 +80,15 @@ Validation Changes
 - ``ValidationInterface`` has been changed. Implemented classes should likewise add the methods and the parameters so as not to break LSP. See :ref:`v430-validation-changes` for details.
 - The return value of  ``Validation::loadRuleGroup()`` has been changed ``null`` to ``[]`` when the ``$group`` is empty. Update the code if you depend on the behavior.
 
+Time Fixes
+==========
+
+Due to bug fixes, some methods in :doc:`Time <../libraries/time>` has been changed the behavior from mutable to immutable. See :ref:`ChangeLog <v430-time-fix>` for details.
```

Comment:
I think the other issue is that now `$time = Time::now(); $time instanceof DateTime` will fail:
```suggestion
Due to bug fixes, some methods in :doc:`Time <../libraries/time>` have changed from mutable behavior to immutable; ``Time`` now extends ``DateTimeImmutable``. See :ref:`ChangeLog <v430-time-fix>` for details.
```

Reply:
Added explanation code.